apps/parsley/src/gql/queries/repo-filters.graphql

```diff
@@ -0,0 +1,12 @@
+query RepoFilters($repoRefId: String!) {
+  repoSettings(repoId: $repoRefId) {
```

Comment:
one question i have is regarding permissions -- `repoSettings.projectRef.parsleyFilters` (requires project settings permissions) is stricter than the `project.parsleyFilters` (requires only basic task permissions). will that be a problem?

Reply:
@minnakt I think this will probably be an issue. What do you think about temporarily using the task permission for the `repoSettings` query and then sorting out the permission further in a followup ticket like we do for the project query in [DEVPROD-6882](https://jira.mongodb.org/browse/DEVPROD-6882)? `RepoSettingsInput` requires project settings permissions so at least writing is secure.

Reply:
we could also keep `repoSettings` query locked down and make a new query like `repo` that just returns the reporef (instead of returning all the settings too)

it would be similar to how we have `projectSettings` query but also `project` query. what do you think 👀

Reply:
@minnakt that sounds good to me! I will make a note in [DEVPROD-6882](https://jira.mongodb.org/browse/DEVPROD-6882) to update the permission for the `repo` query.